### ISSUE TITLE
Monkey King Aghanims doesn't work properly

### ISSUE THREAD

**Issue by nebojsa7** [ValveSoftware/Dota2-Gameplay#12137]
### Item name

Aghanim Scepter

### Description

Monkey King's Aghanims doesn't spawn monkeys properly. Throughout the game it stop spawning other monkeys. Please check the gameplay video. Started malfunctioning from the timestamp 21:30 

### Example Match ID and Timestamp

7348339696 - 21:30

**Comment by danielcarnine** [comment]
I'm adding another Match ID and Timestamp
7348397047 - 25:51

We tried having him move it to the ground, backpack and stash. We lost so we couldn't try sell and buy or eat

**Comment by Avarise831** [comment]
#12064

**Comment by pablopeguero** [comment]
yup. its been 3-4 days for me now of not being able to play my best mid lane hero

**Comment by shaileshgarg** [comment]
https://github.com/ValveSoftware/Dota2-Gameplay/issues/12089

**Comment by Gesuling** [comment]
Yes, they should fix this bug. MK scepter clone spawns only when wukong ring is on cd.

**Comment by powerofgreed** [comment]
https://github.com/ValveSoftware/Dota2-Gameplay/issues/12064
